roles/edpm_ovn/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # defaults file for edpm_ovn
 edpm_ovn_config_src: /var/lib/openstack/configs/ovn
-edpm_ovn_neutron_config_dir: "/var/lib/config-data/ansible-generated/neutron"
+edpm_ovn_neutron_config_dir: "/var/lib/config-data/ansible-generated/neutron_ovn_metadata_agent"
```

Comment:
(No action required) I think we could also rename the variable to accommodate for later introduction of dhcp and sriov and other agents, but I guess it can be postponed to later.